queries/clojure/highlights.scm

```diff
@@ -100,11 +100,17 @@
  (#lua-match? @type "^[^/]+[.][^/]*$"))
 
 ; Interop
-((sym_lit) @method
- (#lua-match? @method "^[.][^-]"))
-((sym_lit) @field
- (#lua-match? @field "^%.-"))
-((sym_lit) @field
+(list_lit
+ .
+ (sym_lit) @method
```

Comment:
Can I get some docs on how these methods/fields in clojure work?

And why is it necessary to limit it to the only the first `sym_lit` of each list

Reply:
I'll remove this change, put it in a different PR.